tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": ["es2017","dom","webworker"],
+      "allowJs": false,
+      "skipLibCheck": true,
+      "allowSyntheticDefaultImports": true,
+      "strict": true,
+      "module": "esnext",
+      "moduleResolution": "node",
+      "resolveJsonModule": true,
+      "jsx": "react"
+    },
+    "exclude": ["./node_modules"]
+}
```

Comment:
Did you look at what was in the SmartCycle-UI tsconfig.json file and change the configuration or is this a default file?